docs/development.md

```diff
@@ -27,6 +28,23 @@ help you get started.
 
 ## Prerequisites
 
+### Binaries
+
+The federation deployment depends on `kubebuilder`, `etcd`, `kubectl`, and
+`kube-apiserver` >= v1.13 being installed in the path. The `kubebuilder`
+([v1.0.8](https://github.com/kubernetes-sigs/kubebuilder/releases/tag/v1.0.8)
+as of this writing) release packages all of these dependencies together.
+
+These binaries can be installed via the `download-binaries.sh` script, which
+downloads them to `./bin`:
+
+```bash
+git clone https://github.com/kubernetes-sigs/federation-v2.git $GOPATH/src/github.com/kubernetes-sigs/federation-v2
```

Comment:
I don't think these 2 lines belong here.  Checking out the repo is something a developer should be able to figure out.